frontend/view-edit-lead/lead-home/lead-metadata.component.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import dayjs from "dayjs";
+
+export default function LeadMetadata(props) {
```

Comment:
Dumb typescript question.

Would it be better to type here and export or will it detect the typings from above?

What's better typings at the bottom of the component tree or typings at a higher level? I assume it's better at the bottom of the tree but I'm not really sure.

Reply:
best to type the component props:

```suggestion
export default function LeadMetadata(props: LeadMetadataProps) {
```